source/brailleDisplayDrivers/seika.py

```diff
@@ -96,18 +97,18 @@ def terminate(self):
 			self._ser.close()
 
 	def display(self, cells: List[int]):
+		# add padding so total length is 1 + numberOfStatusCells + numberOfRegularCells
+		cellPadding: bytes = bytes(self.numCells - len(cells))
+		writeBytes: List[bytes] = [self.sendHeader, ]
 		# every transmitted line consists of the preamble SEIKA_SENDHEADER and the Cells
 		if self.numCells==80:
-			lineBytes = b"".join([
-				b"\xff\xff\x73\x38\x30\x00\x00\x00",
-				bytes(cells)
-			])
+			lineBytes: bytes = self.sendHeader + bytes(cells) + cellPadding
 		else:
-			lineBytes = b"".join([
-				self.s40,
-				b"\0",
-				bytes(cells)
-			])
+			for cell in cells:
+				writeBytes.append(b"\x00")
+				writeBytes.append(intToByte(cell))
+			lineBytes = b"".join(writeBytes) + cellPadding
+		# log.info("senden....{p}".format(p=len(lineBytes)))

```

Comment:
Please don't leave this line commented out. If it is useful  to have this info logged move it  to a lover log level, otherwise just delete it.